mla/threeml/models.py

```diff
@@ -123,7 +123,7 @@ def _init_sob_ratio(self, *args, **kwargs) -> None:
 
         # Normalize histograms by dec band
         with warnings.catch_warnings():  # divide zero warnings
-            warnings.simplefilter("ignore")  # we don't use those bins
+            warnings.simplefilter('ignore')  # we don't use those bins
```

Comment:
please use np.errstate instead. look at __post_init__() in I3EventModel for an example

Reply:
I fixed this in a new commit to this branch